baybe/constraints/discrete.py

```diff
@@ -159,7 +159,9 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         # with a dummy value to cause degeneracy.
         censored_data = data.copy()
         for k, _ in enumerate(self.parameters):
-            censored_data.loc[
+            # .loc assignments are not supported by mypy + pandas-stubs yet
+            # See https://github.com/pandas-dev/pandas-stubs/issues/572
+            censored_data.loc[  # type: ignore[call-overload]
```

Comment:
Wow, pandas is just absolutely bad when it comes to types ...

Reply:
+1 :D